tests/tests/namespace/test-config-maps.spec.js

```diff
@@ -108,4 +110,18 @@ context('Test Config Maps', () => {
       .contains('.fd-layout-panel', ENTRY_KEY2)
       .contains(ENTRY_VALUE2);
   });
+
+  it('Delete Config Maps', () => {
+    cy.getLeftNav()
+      .contains('Config Maps')
+      .click();
+
+    deleteFromGenericList(CLONE_NAME);
+
+    cy.getLeftNav()
+      .contains('Config Maps')
+      .click();
+
+    deleteFromGenericList(CONFIG_MAP_NAME);
```

Comment:
I question if we should delete all namespace-wide resources. It would be nice to test it for one of them to check the mechanics, but those resources will be deleted together with a namespace, so there's no need to run more tests that are not very useful.